src/components/container/ConnectEhr/ConnectEhr.tsx

```diff
@@ -95,26 +102,52 @@ export default function (props: ConnectEhrProps) {
 		return null;
 	}
 
+	if (props.hideWhenConnected && connected) {
+		return null;
+	}
+
+	let defaultTitle = language('connect-ehr-title-prefix') + language('connect-ehr-title-providers') + language('connect-ehr-title-divider') + language('connect-ehr-title-health-plans');
+	let title = props.title || defaultTitle;
+	let connectedText = props.connectedText || language("connect-ehr-text-connected").replace("@@PROJECT_NAME@@", projectName || "");
+	let notConnectedText = props.notConnectedText || language("connect-ehr-text").replace("@@PROJECT_NAME@@", projectName || "");
+	let text = (connected ? connectedText : notConnectedText);
+
+	let headerVariant = props.variant || "large";
+
+	let content = <>
+		<Title autosizeImage defaultMargin order={headerVariant == "large" ? 2 : 3} imageAlignment={headerVariant == "large" ? "top" : "left"} image={<img src={ConnectEHR} />}>{title}</Title>
+		<TextBlock>
+			{connected
+				? <>
```

Comment:
What are the empty brackets <> doing?

Reply:
This is common react syntax when you need a single root node but don't want to actually create a div.